lib/screens/elevator.ex

```diff
@@ -0,0 +1,45 @@
+defmodule Screens.Elevator do
+  @moduledoc """
+  Exposes hand-authored data about elevator accessibility that is currently owned by the Screens
+  team and not (yet?) available in GTFS.
+  """
+
+  alias Screens.Facilities.Facility
+
+  @enforce_keys ~w[id redundancy]a
+  defstruct @enforce_keys
+
+  @type t :: %__MODULE__{id: Facility.id(), redundancy: redundancy()}
+
+  @type redundancy ::
+          :nearby
+          | :in_station
+          | {:different_station, summary :: String.t()}
+          | {:contact, summary :: String.t()}
+
+  @data :screens
+        |> :code.priv_dir()
+        |> Path.join("elevators.json")
+        |> File.read!()
+        |> Jason.decode!()
+
+  @callback get(Facility.id()) :: t() | nil
+  def get(id) do
+    case Map.get(@data, id) do
+      %{"redundancy" => 1} ->
+        %__MODULE__{id: id, redundancy: :nearby}
+
+      %{"redundancy" => 2} ->
```

Comment:
Category 2 should also be returning summary text as well shouldn't it? At least according to the task that this is a prerequisite for mentioned in the PR description.

Reply:
I'm conceiving of this module as purely an interface to the data in the spreadsheet — "rules" apart from the raw data, like "when the redundancy category is 2, always use this specific text" would be implemented in the widget generator. I decided to draw the line here because of the rule where the summary text for _all_ categories changes if any of the "backup elevators" for a given elevator are down, which depends on data from outside the spreadsheet (active alerts).